From a25642d3dbafe88a7c61f2782102a0c811451f0e Mon Sep 17 00:00:00 2001
From: Chandresh Pitty <Chandresh_Pitty@cable.comcast.com>
Date: Thu, 28 May 2015 12:40:37 -0400
Subject: [PATCH 10/10] RDK-5414 : GStreamer 1.0 QT Base 5.1 BB file Mods
 Reason for change: Mods to ensure that Qt Base 5.1.1 is built with GStreamer
 1.0 Test Procedure: Test Live/VOD/DVR playback and trickplays with GStreamer
 1.0 enabled Risks: Major Signed-off-by: Chandresh Pitty
 <Chandresh_Pitty@cable.comcast.com>

Change-Id: I559543ace40ec671b87a931ffc8eac6ecd7617bf
---
 recipes-qt/qt5/qtbase-5.1.1.inc | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/recipes-qt/qt5/qtbase-5.1.1.inc b/recipes-qt/qt5/qtbase-5.1.1.inc
index 12d7f7c..ddb8f62 100644
--- a/recipes-qt/qt5/qtbase-5.1.1.inc
+++ b/recipes-qt/qt5/qtbase-5.1.1.inc
@@ -107,7 +107,8 @@ PACKAGECONFIG[iconv] = "-iconv,-no-iconv,virtual/libiconv"
 PACKAGECONFIG[xkb] = "-xkb,-no-xkb,libxkbcommon"
 PACKAGECONFIG[evdev] = "-evdev,-no-evdev"
 # depends on glib
-PACKAGECONFIG[gstreamer] = "-gstreamer,-no-gstreamer,gstreamer gst-plugins-base"
+PACKAGECONFIG[gstreamer010] = "-gstreamer,-no-gstreamer,gstreamer gst-plugins-base"
+PACKAGECONFIG[gstreamer] = "-gstreamer,-no-gstreamer,gstreamer1.0 gstreamer1.0-plugins-base"
 PACKAGECONFIG[fontconfig] = "-fontconfig,-no-fontconfig,fontconfig"
 PACKAGECONFIG[gtkstyle] = "-gtkstyle,-no-gtkstyle,gtk+"
 PACKAGECONFIG[directfb] = "-directfb,-no-directfb,directfb"
-- 
2.4.2

